config/initializers/vcap_services.rb

```diff
@@ -0,0 +1,2 @@
+require "vcap_parser"
+VcapParser.load_service_environment_variables!
```

Comment:
It would be great but not essential to have a comment here that explains what a `vcap` is to those that are new to it, and when the reader needs to worry about it.

Something like: 

> VCAP services is a Cloud Foundry mechanism that helps provision application environment variables to allow the app to run on Government Platform as a Service (GPaaS).

Reply:
sure.